jobs/openshift4/cluster/create/Jenkinsfile

```diff
@@ -0,0 +1,180 @@
+                        mkdir ~/.aws
+                        touch ~/.aws/credentials
+                        set +x # suppress output of the next shell command
+                        echo "[default]\naws_access_key_id=${AWS_ACCESS_KEY_ID}\naws_secret_access_key=${AWS_SECRET_ACCESS_KEY}" > ~/.aws/credentials
+                        set -x
+                    """
+                )
+            } // stage
+            
+            stage("Prepare the repo to store created cluster information") {
+                dir("rhmi-jenkins") {
+                    git branch: "master", url: "https://gitlab.cee.redhat.com/integreatly-qe/rhmi-jenkins.git"
+                    def dirExists = fileExists(clusterInfoDirectory)
+                    if (dirExists) {
+                        sh "cd clusters && rm -rf ${clusterName}"
+                    } 
+                    sh "cd clusters && mkdir ${clusterName}"
+                }
+            } // stage
+            
+            stage("Format the cluster creation template") {
+                dir("rhmi-jenkins") {
+                    sh(
+                        returnStdout: false,
+                        script: """
+                            cp templates/install-config.yaml ${clusterInfoDirectory}
+                            sed -i 's/CLUSTER_NAME/${clusterName}/g' ${clusterInfoDirectory}/install-config.yaml
+                            sed -i 's/COMPUTE_REPLICAS/${COMPUTE_REPLICAS}/g' ${clusterInfoDirectory}/install-config.yaml
+                            sed -i 's/CONTROL_PLANE_REPLICAS/${CONTROL_PLANE_REPLICAS}/g' ${clusterInfoDirectory}/install-config.yaml
+                            sed -i 's/AWS_REGION/${awsRegion}/g' ${clusterInfoDirectory}/install-config.yaml
+                            sed -i 's/BASE_DOMAIN/${baseDomain}/g' ${clusterInfoDirectory}/install-config.yaml
+                            echo 'Creating OpenShift 4 cluster with the following configuration:'
+                            cat ${clusterInfoDirectory}/install-config.yaml
+                        """
+                    )
+                }
+            } // stage
+            
+            stage("Create OpenShift 4 cluster") {
+                dir(".") {
+                    rootWorkspaceDir = sh (
+                        script: "pwd",
+                        returnStdout: true
+                    ).trim()
+                    installConfigDirectory = rootWorkspaceDir + "/rhmi-jenkins/" + clusterInfoDirectory + "/"
+                    sh "./openshift-install create cluster --dir=${installConfigDirectory} --log-level=${LOG_LEVEL} 2>&1 | tee log.file"
+                    String clusterCreationOutput = readFile("log.file")
+                    sh "mv log.file ${installConfigDirectory}"
+                }
+            } // stage
+            
+            stage("Push config files to GitLab") {
+                withCredentials([string(credentialsId: "qe-gitlab-access-token", variable: "GITLAB_ACCESS_TOKEN")]) {
```

Comment:
Is this a new credential (qe-gitlab-access-token)? Has it been added to the integreatly-dev config repo?

Reply:
It will be along with aws credentials